ProjectEureka/project-eureka-flutter
# PE-56 | Implemented UI for Login

What I Did: 

- Implemented the UI specified on Figma (with a few slight changes) for **Login**, **Sign Up**, and **Forgot Password.**

How to Test: 

1. Sign up. Try using an actual email so that you can check to see if a**reset** password link actually sends to your email. Check to see if validation works.
2. Sign in with the credentials you signed up with. Again, check to see if validation works. 
3. Click on **Forgot password?** and do what it says.
